src/desktop/analytics/main_layout.js

```diff
@@ -24,9 +24,9 @@ if (pageType == "artwork") {
   }
 }
 
-// We exclude 'orders' routes from analytics.page calls because they're already
+// We exclude these routes from analytics.page calls because they're already
 // taken care of in another place.
-const excludedRoutes = ["orders"]
+const excludedRoutes = ["orders", "new-artwork"]
```

Comment:
Maybe add a `FIXME: remove new-artwork when fully deployed` comment and / or something about where its being called from Reaction? "In another place" is a bit vague.